scripts/kind-e2e/e2e.sh

```diff
@@ -51,6 +51,7 @@ with_context cluster3 subctl_install_subm
 
 with_context cluster2 deploy_resource "${RESOURCES_DIR}/netshoot.yaml"
 with_context cluster3 deploy_resource "${RESOURCES_DIR}/nginx-demo.yaml"
+[[ "${lighthouse}" = "true" ]] && with_context cluster3 kubectl apply -f "${DAPPER_SOURCE}/scripts/kind-e2e/nginx-demo-export.yaml"
```

Comment:
used kubectl apply -f because deploy_resource will check for rollout status... and we don't have that on service export.